docs/source/dev/sphinx.rst

```diff
@@ -39,6 +39,20 @@ The following requirements need to be installed (once) to build our documentatio
     # python tools & style theme
     pip install -r requirements.txt # --user
 
+In order to not break any of your existing Python configurations, you can also create a new environment that you only use for building the documentation.
+Since it is possible to install doxygen with conda, the following method employs this way.
```

Comment:
```suggestion
Since it is possible to install doxygen with conda, the following demonstrates this.
```

Reply:
Implemented your suggestions in one commit